### ISSUE TITLE
problem running jasp on Dell XPS (ice-lake video) ubuntu 20

### ISSUE THREAD

**Issue by Gaoithe** [jasp-stats/jasp-issues#998]
* JASP version:0.13.1
* OS name and version:Linux ubuntu 20.04 
*** Analysis: jasp runs but GUI is corrupted, Dell XPS ice-lake graphics, jasp installed with flatpak,** 
* Bug description: 
```
XPS-13:~$ flatpak run org.jaspstats.JASP --safeGraphics 2>&1 |tee run_jasp_safe_DellXPS_icelake_err.log
Gtk-Message: 19:57:22.882: Failed to load module "canberra-gtk-module"
Gtk-Message: 19:57:22.882: Failed to load module "canberra-gtk-module"
Qt: Session management error: None of the authentication protocols specified are supported
qrc:/components/JASP/Widgets/ComputeColumnWindow.qml:330:2: QML Dialog: Binding loop detected for property "implicitWidth"
qrc:/components/JASP/Widgets/ComputeColumnWindow.qml:330:2: QML Dialog: Binding loop detected for property "implicitWidth"
qrc:/components/JASP/Widgets/FilterWindow.qml:362:4: QML Dialog: Binding loop detected for property "implicitWidth"
qrc:/components/JASP/Widgets/FilterWindow.qml:362:4: QML Dialog: Binding loop detected for property "implicitWidth"
qrc:/components/JASP/Widgets/FilterWindow.qml:112:4: QML Dialog: Binding loop detected for property "implicitWidth"
qrc:/components/JASP/Widgets/FilterWindow.qml:112:4: QML Dialog: Binding loop detected for property "implicitWidth"
Property 'data'' of object 'QQuickRootItem' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'resources'' of object 'QQuickRootItem' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'states'' of object 'QQuickRootItem' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'transitions'' of object 'QQuickRootItem' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'transformOriginPoint'' of object 'QQuickRootItem' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'transform'' of object 'QQuickRootItem' has no notify signal and is not constant, value updates in HTML will be broken!
Don't know how to handle 'mode', use qRegisterMetaType to register it.
Don't know how to handle 'format', use qRegisterMetaType to register it.
Don't know how to handle 'mirroring', use qRegisterMetaType to register it.
Don't know how to handle '', use qRegisterMetaType to register it.
Property 'data'' of object 'QQuickItem' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'resources'' of object 'QQuickItem' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'states'' of object 'QQuickItem' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'transitions'' of object 'QQuickItem' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'transformOriginPoint'' of object 'QQuickItem' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'transform'' of object 'QQuickItem' has no notify signal and is not constant, value updates in HTML will be broken!
Don't know how to handle '', use qRegisterMetaType to register it.
Property 'data'' of object 'QQuickItem' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'resources'' of object 'QQuickItem' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'states'' of object 'QQuickItem' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'transitions'' of object 'QQuickItem' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'transformOriginPoint'' of object 'QQuickItem' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'transform'' of object 'QQuickItem' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'data'' of object 'SplitView_QMLTYPE_142_QML_163' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'resources'' of object 'SplitView_QMLTYPE_142_QML_163' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'states'' of object 'SplitView_QMLTYPE_142_QML_163' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'transitions'' of object 'SplitView_QMLTYPE_142_QML_163' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'transformOriginPoint'' of object 'SplitView_QMLTYPE_142_QML_163' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'transform'' of object 'SplitView_QMLTYPE_142_QML_163' has no notify signal and is not constant, value updates in HTML will be broken!
Don't know how to handle '', use qRegisterMetaType to register it.
Property 'data'' of object 'QQuickItem' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'resources'' of object 'QQuickItem' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'states'' of object 'QQuickItem' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'transitions'' of object 'QQuickItem' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'transformOriginPoint'' of object 'QQuickItem' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'transform'' of object 'QQuickItem' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'data'' of object 'QQuickRectangle' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'resources'' of object 'QQuickRectangle' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'states'' of object 'QQuickRectangle' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'transitions'' of object 'QQuickRectangle' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'transformOriginPoint'' of object 'QQuickRectangle' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'transform'' of object 'QQuickRectangle' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'gradient'' of object 'QQuickRectangle' has no notify signal and is not constant, value updates in HTML will be broken!
Don't know how to handle '', use qRegisterMetaType to register it.
Property 'data'' of object 'QQuickWebEngineView' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'resources'' of object 'QQuickWebEngineView' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'states'' of object 'QQuickWebEngineView' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'transitions'' of object 'QQuickWebEngineView' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'transformOriginPoint'' of object 'QQuickWebEngineView' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'transform'' of object 'QQuickWebEngineView' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'userScripts'' of object 'QQuickWebEngineProfile' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'data'' of object 'QQuickWebEngineHistory' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'resources'' of object 'QQuickWebEngineHistory' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'states'' of object 'QQuickWebEngineHistory' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'transitions'' of object 'QQuickWebEngineHistory' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'transformOriginPoint'' of object 'QQuickWebEngineHistory' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'transform'' of object 'QQuickWebEngineHistory' has no notify signal and is not constant, value updates in HTML will be broken!
Don't know how to handle '', use qRegisterMetaType to register it.
Property 'transports'' of object 'QQmlWebChannel' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'registeredObjects'' of object 'QQmlWebChannel' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'userScripts'' of object 'QQuickWebEngineView' has no notify signal and is not constant, value updates in HTML will be broken!
Don't know how to handle '', use qRegisterMetaType to register it.
Property 'data'' of object 'QQuickItem_QML_165' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'resources'' of object 'QQuickItem_QML_165' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'states'' of object 'QQuickItem_QML_165' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'transitions'' of object 'QQuickItem_QML_165' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'transformOriginPoint'' of object 'QQuickItem_QML_165' has no notify signal and is not constant, value updates in HTML will be broken!
Property 'transform'' of object 'QQuickItem_QML_165' has no notify signal and is not constant, value updates in HTML will be broken!
Don't know how to handle '', use qRegisterMetaType to register it.
R_HOME: /app/lib64/R/
R_HOME: /app/lib64/R/
R_HOME: /app/lib64/R# same result
R_HOME: /app/lib64/R/
```

glxinfo shows . . . 
```
Extended renderer info (GLX_MESA_query_renderer):
    Vendor: Intel (0x8086)
    Device: Mesa Intel(R) Iris(R) Plus Graphics (ICL GT2) (0x8a52)
    Version: 20.0.8
    Accelerated: yes
    Video memory: 3072MB
    Unified memory: yes
    Preferred profile: core (0x1)
    Max core profile version: 4.6
    Max compat profile version: 4.6
    Max GLES1 profile version: 1.1
    Max GLES[23] profile version: 3.2
[glxinfo_Dell_XPS.log](https://github.com/jasp-stats/jasp-issues/files/5324160/glxinfo_Dell_XPS.log)
```

* Expected behaviour:

#### Steps to reproduce:
1. Start jasp:
`flatpak run org.jaspstats.JASP --safeGraphics`
or `flatpak run org.jaspstats.JASP # same result`

![Screenshot from 2020-10-04 19-56-26](https://user-images.githubusercontent.com/3024997/95024891-cb837200-067d-11eb-9a08-b4dc2b8e4ea7.png)

**Comment by JorisGoosen** [comment]
Thank you for the detailed bugreport!

Too bad that the usual solution I have for problems caused by `Intel Iris` is to turn `safeGraphics` on, which you have already done...

Is it possible you also have a dedicated GPU next to the intel one?

For fullness:
- Are you on the latest flatpak from [the author](https://launchpad.net/~alexlarsson/+archive/ubuntu/flatpak)?
- If you weren't or not, have you made sure to run `flatpak update` ?

**Comment by Gaoithe** [comment]
> Thank you for the detailed bugreport!
Very welcome, thank you for responding. Apologies now big lag in my response.
We are running the Jasp under windows (dual boot) so not blocked by this problem but of course would be nice to solve it.

..
> Is it possible you also have a dedicated GPU next to the intel one?
It's a Dell XPS 13, no add on GPU (and no built in GPU like the XPS 15), 
So this is with just the built in Intel graphics.
 
> For fullness:
> 
> * Are you on the latest flatpak from [the author](https://launchpad.net/~alexlarsson/+archive/ubuntu/flatpak)?
> * If you weren't or not, have you made sure to run `flatpak update` ?
Ahhh, ohh. Ok, I will poke around in this area when next on machine.
Apologies, I expect more lag.

**Comment by JorisGoosen** [comment]
Well, I'm not in any hurry, glad to hear you have backup option.

It is too bad intel manages to always break software with their GPUs... These problems rarely occur with AMD/NVIDIA, but ah well...

**Comment by juliuspf** [comment]
@Gaoithe, does the issue persist with the latest JASP version?
